sdk/src/main/java/com/meniga/sdk/models/feed/MenigaAccountEvent.java

```diff
@@ -134,6 +133,8 @@ public void writeToParcel(Parcel dest, int flags) {
 		dest.writeSerializable(this.date);
 		dest.writeString(this.title);
 		dest.writeString(this.body);
+		dest.writeInt(this.eventTypeIdentifier == null ? -1 : this.eventTypeIdentifier.ordinal());
+		dest.writeString(this.topicName);
```

Comment:
The whole class may be converted to a data class with `@Parcelize` and `var messageData: MenigaAccountEventData?` (as this one is mutable by the converter) so we don't have to think about this code.

Reply:
I've changed it into a Kotlin class.